pkg/controller/operconfig/migration.go

```diff
@@ -32,6 +43,84 @@ func migrateEgressFirewallCRs(ctx context.Context, operConfig *operv1.Network, c
 	return nil
 }
 
+func enableMulticastOVN(ctx context.Context, client cnoclient.Client) error {
+	// 1. query for netnamespaces
+	nnsList, err := client.Default().Dynamic().Resource(gvrNetnamespace).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	// 2. iterate through netnamespaces
+	//    - any with multicast-enabled="true" annotation will cause an update to the corresponding
+	//      namespace to add the necessary OVN annotation
+	for _, nns := range nnsList.Items {
+		if nns.Object["metadata"].(map[string]interface{})["annotations"] == nil {
+			continue
+		}
+		multicastAnnotation := nns.Object["metadata"].(map[string]interface{})["annotations"].(map[string]interface{})["netnamespace.network.openshift.io/multicast-enabled"]
+		if multicastAnnotation == "true" {
+			// first update namespace to have the same annotation
+			nspStr := fmt.Sprintf("%v", nns.Object["netname"])
+			namespaceObj, err := client.Default().Kubernetes().CoreV1().Namespaces().Get(ctx, nspStr, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			namespaceObj.Annotations["k8s.ovn.org/multicast-enabled"] = "true"
+
+			if err := apply.ApplyObject(ctx, client, namespaceObj, ""); err != nil {
```

Comment:
You should not use `apply.ApplyObject` for this; that is for creating/updating objects _that CNO owns_, and may do things you don't want, like updating `ManagedFields`. You should just update the object normally. (Except that you have to use `RetryOnConflict`.)